src/call/mod.rs

```diff
@@ -274,6 +224,15 @@ impl BatchContext {
         self.ctx
     }
 
+    pub fn take_recv_message(&self) -> Option<GrpcByteBuffer> {
+        let ptr = unsafe { grpc_sys::grpcwrap_batch_context_take_recv_message(self.ctx) };
```

Comment:
Where will the pointer will be de-alloc?

Reply:
It's wrapped into a `GrpcByteBuffer`, and will be de-alloc by
```rust
impl Drop for GrpcByteBuffer {
    fn drop(&mut self) {
        unsafe { grpc_byte_buffer_destroy(self.raw) }
    }
}
```